graderx/graders/unittest_graders/pytest/lib/submissions_extraction.py

```diff
@@ -0,0 +1,53 @@
+import shutil
+import os
+import patoolib
+
+
+def clean_directory(dir):
+    files = dir.glob('*')
+    for f in files:
+        if f.is_dir():
+            shutil.rmtree(str(f))
+        else:
+            f.unlink()
+
+
+def extract_file(file_path, verbosity=-1):
+    patoolib.extract_archive(
+        file_path, outdir=file_path.parent, verbosity=verbosity)
+    os.remove(file_path)
+
+
+def extract_submissions(dest_directory, submissions_file,  verbosity=0):
```

Comment:
why do we have 2 files or sumbissions_extraction?
`graderx/graders/stdout_graders/c/lib/submissions_extraction.py`
`graderx/graders/unittest_graders/pytest/lib/submissions_extraction.py`